### DISCUSSION TITLE
Thank you for fixing the memory leak issue! 🙏

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#571] [Q&A]
Hi @pgundlach 

thank you very much for fixing the memory leak issue, I was opening a bug report but 4.17.21 resolved it!
It was really driving me crazy cause I couldn't rendering anything on my crapbook... 😵

Thanks again! 🙏🙏🙏

**Comment by pgundlach** [comment]
Do you remember which version had the memory leak? Which operating system?

**Reply by Cicorione** [reply]
Hi Patrick,

I am pretty sure this occurred with `4.17.19` and `4.17.20` with `sdluatex` quickly filling all the `RAM` and the `SWAP` memory forcing the Linux Kernel to shutdown the process.

OS Specs:
```
Distributor ID:	Devuan
Description:	Devuan GNU/Linux 5 (daedalus)
Release:	5
Codename:	daedalus
Linux a11bk 6.1.0-18-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.76-1 (2024-02-01) x86_64               free -m
               total        used        free      shared  buff/cache   available
Mem:            3737        2704         269         120        1119        1033
Swap:            975           4         971
```

With this updated 4.17.21 not only the memory leak is disappeared but the rendering time is visibly faster!

**Comment by pgundlach** [comment]
Very interesting, I cannot reproduce the problem (fixed anyway now) and I don't know what would have fixed the problem.

**Reply by Cicorione** [reply]
Very true...
